src/ui/molecules/MultiFileUpload.tsx

```diff
@@ -36,6 +36,7 @@ interface FileItemProps extends FileState {
 
 const FileItem = ({ uploadInProgress, error, fileStored, onDelete, disableDelete }: FileItemProps): JSX.Element => {
     const { t } = useTranslation('ui');
+    const progress = uploadInProgress ? uploadInProgress.progress : null;
```

Comment:
Can we add a unit test for this or update the tests that are there?

Reply:
The tests we updated in the previous PR